src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/factory/TemporaryAccommodationPremisesEntityFactory.kt

```diff
@@ -94,6 +95,10 @@ class TemporaryAccommodationPremisesEntityFactory : Factory<TemporaryAccommodati
     this.probationDeliveryUnit = probationDeliveryUnit
   }
 
+  fun withTurnaroundWorkingDayCount(turnaroundWorkingDayCount: Yielded<Int>) = apply {
+    this.turnaroundWorkingDayCount = turnaroundWorkingDayCount
+  }
```

Comment:
I think this would be more consistent as:

```suggestion
  fun withTurnaroundWorkingDayCount(turnaroundWorkingDayCount: Int) = apply {
    this.turnaroundWorkingDayCount = { turnaroundWorkingDayCount }
  }
```